src/Sylius/Behat/Context/Api/Admin/ManagingPaymentsContext.php

```diff
@@ -26,6 +26,8 @@
 
 final class ManagingPaymentsContext implements Context
 {
+    private const RESOURCE_PAYMENTS = 'payments';
```

Comment:
sometimes you are using more detailed names of resources, sometimes you use just resource: 
<img width="567" alt="image" src="https://user-images.githubusercontent.com/6213903/161989864-dbd5e373-6fb7-43b2-8de1-eb7f0208059e.png">

Would be nice to be coherent among all contexts with it. Perhaps it would make sense to just extract these consts to separate object? How often it happens, that we are declaring the same value as const?

Reply:
I was coherent :D but in a different way :). if context use a single resource it was `resource` if it use more resources I declared the resource with a name :)

but I like the idea of separate object with all resource names (especially when for `shop` and `admin` we have same names`

Reply:
I was expecting such an answer - but either I'm missing something or you are wrong.

"if context use a single resource it was resource if it use more resources I declared the resource with a name :)"

The exact const that I commented disproves your sentence:
<img width="414" alt="image" src="https://user-images.githubusercontent.com/6213903/162223074-a7c84d2f-c131-4e94-b0d0-8a3600455c4d.png">

There is only single resource, yet it is with a name.

It is also like this in ProductVariant: 
<img width="818" alt="image" src="https://user-images.githubusercontent.com/6213903/162223910-733af6b6-a0c1-42d0-a205-d00d5832548f.png">

Reply:
I've created the rule but also I've doesn't follow it :(
you're right this inconsistency was made by mistake.
but I've rethought it and decided to remove these consts at all :)